internal/commands/app/create_test.go

```diff
@@ -504,23 +511,22 @@ Check out your app: cd ./remote-app && realm-cli app describe
 		profile.SetRealmBaseURL("http://localhost:8080")
 
 		out, ui := mock.NewUI()
-
 		testApp := realm.App{
-			ID:          "789",
+			ID:          "456",
 			GroupID:     "123",
 			ClientAppID: "bitcoin-miner-abcde",
 			Name:        "bitcoin-miner",
 		}
 
-		backendZipPkg, err := zip.OpenReader("testdata/bitcoin-miner.zip")
+		backendZipPkg, err := zip.OpenReader("testdata/project.zip")
 		assert.Nil(t, err)
 
 		client := mock.RealmClient{
 			FindAppsFn: func(filter realm.AppFilter) ([]realm.App, error) {
 				return []realm.App{}, nil
 			},
 			ExportFn: func(groupID, appID string, req realm.ExportRequest) (string, *zip.Reader, error) {
-				return "", &backendZipPkg.Reader, err
```

Comment:
👀

Reply:
this err _is_ harmless to return, but there's no need

Reply:
just saying, named errors would never let such a thing occur :bushes: